### DISCUSSION TITLE
Migrazione da Micro SD a  HD SSD

### DISCUSSION THREAD

**Discussion by toniobr1976** [indomus/forum#854] [HUB: Home Assistant]
Salve,
per migrare da Micro SD Home Assistant 32bit a HD SSD HA 64bit ho eseguito la seguente procedura:

-backup completo HA con la funzione integrata in supervisor
-salvataggio backup su PC
-poweroff HA
-rimozione SD
-download ultima versione di HA dal sito ufficiale
-caricamento scrittura di HA su SSD con balenaEtcher
-collegato HD alla porta nera del Rasberry
-avviato ... attesa 
-caricamento del precedente backup da supervisor
FINE con esito OK

Questa è la semplice procedura che mi ha consigliato un amico in quanto in rete ho trovato tante informazioni confuse e complesse tipo aprire delle connessioni SSh, utilizzare TRIM ecc ecc.
Come mai? 
la condivido magari potrà tornare utile a qualcuno.

**Comment by tempod** [comment]
anche io volevo migrare ad un ssd.
la cosa che mi frena è dover riconfigurare qualcosa.
hai avuto problemi in questo senso?

che componenti hai usato?

**Reply by toniobr1976** [reply]
Solo esphome l ho dovuto disinstallare e reinstallare

**Comment by StefanoA70** [comment]
vale allo stesso modo per Pi4B e Pi3B o ci sono delle differenze ?

**Reply by toniobr1976** [reply]
non so dirti

**Reply by StefanoA70** [reply]
tu quale hai usato?

**Comment by barbatrukko** [comment]
> Salve, per migrare da Micro SD Home Assistant 32bit a HD SSD HA 64bit ho eseguito la seguente procedura:
> 
> -backup completo HA con la funzione integrata in supervisor -salvataggio backup su PC -poweroff HA -rimozione SD -download ultima versione di HA dal sito ufficiale -caricamento scrittura di HA su SSD con balenaEtcher -collegato HD alla porta nera del Rasberry -avviato ... attesa -caricamento del precedente backup da supervisor FINE con esito OK
> 
> Questa è la semplice procedura che mi ha consigliato un amico in quanto in rete ho trovato tante informazioni confuse e complesse tipo aprire delle connessioni SSh, utilizzare TRIM ecc ecc. Come mai? la condivido magari potrà tornare utile a qualcuno.

che marca di ssd/case esterno hai usato?
Stiamo parlando di RasPi4 ?

**Reply by barbatrukko** [reply]
perché io con un crucial BX ed un case economico sono proprio a limite ed alle volte al boot fa fatica...
